webhook/webhook.go

```diff
@@ -0,0 +1,42 @@
+package webhook
+
+type Webhook struct {
+  client *http.Client
+}
+
+//TODO
+
+
+func (whook *Webhook) Send(server string, secret []byte, data interface{}) error {
```

Comment:
Super minor nit - I'd rename this to `wh` instead of `whook`. Keeps it cleaner and matches what you have in the `models` package above.